x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_to_case_actions.tsx

```diff
@@ -80,7 +80,9 @@ export const useAddToCaseActions = ({
     onMenuItemClick();
     createCaseFlyout.open({
       attachments: caseAttachments,
-      ...(isTourShown(SecurityStepId.alertsCases) && activeStep === 4
+      // activeStep will be 4 on first render because not yet incremented
+      // if the user closes the flyout without completing the form and comes back, we will be at step 5
```

Comment:
![2a](https://user-images.githubusercontent.com/6935300/198686105-ced740af-ac32-487a-8533-a0400a650f70.gif)
Fixes this issue